src/test/java/org/broadinstitute/consent/http/db/LibraryCardDAOTest.java

```diff
@@ -219,6 +219,20 @@ void testFindLibraryCardByInstitutionId() {
     assertTrue(cardsFromDAO.get(0).getDaaIds().isEmpty());
   }
 
+  @Test
+  void testFindLibraryCardByUserIdInstitutionId() {
+    LibraryCard libraryCard = createLibraryCard();
```

Comment:
To exercise the daa part of the new query, I suggest adding this code:
```
    int dacId = dacDAO.createDac(RandomStringUtils.randomAlphabetic(5), RandomStringUtils.randomAlphabetic(5), RandomStringUtils.randomAlphabetic(5), new Date());
    Instant now = Instant.now();
    int daaId = daaDAO.createDaa(libraryCard.getUserId(), now, libraryCard.getUserId(), now, dacId);
    daaDAO.createDacDaaRelation(dacId, daaId);
    libraryCardDAO.createLibraryCardDaaRelation(libraryCard.getId(), daaId);
```
and then ensuring that `cardsFromDAO.get(0).getDaaIds().isEmpty()` is false